### PR TITLE
[FIX] oca_install_addons: Update apt-get if deb packages must be installed

### PR DESCRIPTION
**Pull request by etobella**
Tries to fix https://github.com/OCA/oca-ci/issues/26

@sbidoul May I ask for your review :smile:

### CODE REVIEW

**Comment by sbidoul** [comment]
Oh look, there is even some test coverage for that part :)

**Comment by etobella** [comment]
Long live to CI Tools :laughing: 

Now it is :green_apple: :green_apple: :green_apple: @sbidoul

File: bin/oca_install_addons [hunk 1]
```diff
@@ -24,5 +24,10 @@ pip list
 echo "addons_path=${ADDONS_PATH},${ADDONS_DIR}" >> ${ODOO_RC}
 cat ${ODOO_RC}
 
-# Install 'deb' external dependencies of all Odoo addons found in path.
-DEBIAN_FRONTEND=noninteractive apt-get install -qq --no-install-recommends $(oca_list_external_dependencies deb)
+deps=$(oca_list_external_dependencies deb)
+if [$deps]; then
+    apt-get update
```
Comment (sbidoul) [inline bin/oca_install_addons:29]:
```suggestion
    apt-get update -qq
```

File: bin/oca_install_addons [hunk 2]
```diff
@@ -24,5 +24,10 @@ pip list
 echo "addons_path=${ADDONS_PATH},${ADDONS_DIR}" >> ${ODOO_RC}
 cat ${ODOO_RC}
 
-# Install 'deb' external dependencies of all Odoo addons found in path.
-DEBIAN_FRONTEND=noninteractive apt-get install -qq --no-install-recommends $(oca_list_external_dependencies deb)
+deps=$(oca_list_external_dependencies deb)
+if [$deps]; then
```
Comment (sbidoul) [inline bin/oca_install_addons:28]: This is not valid bash. Maybe this? Not sure.

```suggestion
if [-n "$deps"]; then
```

**Comment by etobella** [comment]
All changes applied :smile:

**Comment by sbidoul** [comment]
Thanks! The updated images should be published in a few minutes.

**Pull request merged**